Gemfile

```diff
@@ -8,4 +8,6 @@ group :development do
   gem 'guard'
   gem 'guard-rspec'
   gem 'rb-readline'
+  gem 'bump'
```

Comment:
Shouldn't all these gems in the :development group really exist in the specfile as a development_dependency?